### ISSUE TITLE
Uncaught exception 'TypeError' in /var/www/html/lib/MRBS/Form/Field.php at line 134

### ISSUE THREAD

**Issue by stueja** [meeting-room-booking-system/mrbs-code#3732]
**Describe the bug**
Suddenly, getting the below mentioned error when clicking on "Rooms" and "Edit Area":

```
 Uncaught exception 'TypeError' in /var/www/html/lib/MRBS/Form/Field.php at line 134
MRBS\Form\Field::setControlText(): Argument #1 ($text) must be of type string, null given, called in /var/www/html/edit_area.php on line 135
#0 /var/www/html/edit_area.php(135): MRBS\Form\Field->setControlText()
#1 /var/www/html/edit_area.php(785): MRBS\get_fieldset_general()
#2 {main}

MRBS GET: Array
(
)
MRBS POST: Array
(
    [MAX_FILE_SIZE] => 2097152
    [csrf_token] => 09e9f5967e443442a060bff7333769789c0d895f539c8e061c8715a0f5ee66af
    [day] => 1
    [month] => 8
    [year] => 2024
    [area] => 2
)
MRBS SESSION: Array
(
    [last_page] => /admin.php
    [this_page] => /edit_area.php
    [csrf_token] => 09e9f5967e443442a060bff7333769789c0d895f539c8e061c8715a0f5ee66af
    [user] => MRBS\User Object
        (
            [username] => username
            [display_name] => User Name
            [email] => user.name@domain.tld
            [level] => 2
            [data:protected] => Array
                (
                    [id] => 1
                    [password_hash] => somehash
                    [timestamp] => 2024-02-19 20:25:47
                    [last_login] => 1724345140
                    [reset_key_hash] =>
                    [reset_key_expiry] => 0
                )

        )

)

#0 MRBS\generate_backtrace() called at [/var/www/html/functions_error.inc:226]
#1 MRBS\output_error() called at [/var/www/html/functions_error.inc:280]
#2 MRBS\output_exception_error(object) called at [/var/www/html/functions_error.inc:288]
#3 MRBS\exception_handler(object)

Oh je! Leider ist in MRBS ein schwerwiegender Fehler aufgetreten. Bitte konsultieren Sie Ihren Systemadministrator.

```

**To Reproduce**
1. Go to Rooms
2. Click on the edit pencil
3. See error

**Expected behavior**
The area edit page comes up.

**Screenshots**

**Browser details (please complete the following information):**
 - OS: macOS 14.6.1
 - Browser Safari 17.6 and Firefox 129.0.1

**Server details (please complete the following information):**
 - MRBS version 1.11.5
 - Web server: nginx/1.22.1
 - OS: Linux d446d0e82d99 5.15.0-113-generic 123-Ubuntu
 - PHP version: 8.1.20
 - Database used: mariadb 11.4.3
 - database schema:	82
 - version of local database schema	1

**Additional context**

**Comment by campbell-m** [comment]
Now fixed in [5f3a83b](https://github.com/meeting-room-booking-system/mrbs-code/commit/5f3a83bf9005cdd2fa13bbb78695cfaef1559ffb).

**Comment by stueja** [comment]
You burned the midnight oil again. Tested and works, thank you very much!
